docs/features/W3cCredentials.md

```diff
@@ -0,0 +1,633 @@
+
+Familiarity with the Verifiable Presentations Data Model is necessary. Relevant resources can be found at:
+
+- [Verifiable Presentations Data Model](https://www.w3.org/TR/vc-data-model/#presentations)
+
+#### DIF Presentation Format
+
+Understanding the DIF Presentation Format is recommended. Access resources at:
+
+- [DIF Presentation Format](https://identity.foundation/presentation-exchange/)
+
+### Preparing to Issue a Credential
+
+To prepare for credential issuance, the following steps must be taken:
+
+#### VC-DI Context
+
+Ensure that every property key in the document is mappable to an IRI. This requires either the property key to be an IRI by default or to have the shorthand property mapped in the `@context` of the document.
+
+```json
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3id.org/security/data-integrity/v2",
+        {
+            "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"
+        }
+    ]
+}
+```
+
+#### Signature Suite
+
+Select a signature suite for use. VC-DI format currently supports EdDSA signature suites for issuing credentials.
+
+- [`Ed25519Signature2020`](https://w3c.github.io/vc-di-eddsa/#ed25519signature2020-0)
+
+#### DID Method
+
+Choose a DID method for issuing the credential. VC-DI format currently supports the `did:key` method.
+
+##### `did:key`
+
+A `did:key` did is not anchored to a ledger, but embeds the key directly in the identifier part of the did. See the [did:key Method Specification](https://w3c-ccg.github.io/did-method-key/) for more information.
+
+You can create a `did:key` using the `/wallet/did/create` endpoint with the following body.
+
+```jsonc
+{
+  "method": "key",
+  "options": {
+    "key_type": "ed25519"
+  }
+}
+```
+
+### Issue a Credential
+
+The issuance of W3C credentials is facilitated through the `/issue-credential-2.0/send` endpoint. This process adheres to the formats described in [RFC 0809 VC-DI](https://github.com/hyperledger/aries-rfcs/blob/main/features/0809-w3c-data-integrity-credential-attachment/README.md) and utilizes `didcomm` for communication between agents.
+
+To issue a W3C credential, follow these steps:
+
+1. **Prepare the Credential Data:**
+Ensure the credential data conforms to the VC-DI context.
+
+<details>
+<summary>JSON example</summary>
+
+```jsonc
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://w3id.org/security/data-integrity/v2",
+    {
+      "@vocab": "https://www.w3.org/ns/credentials/issuer-dependent#"
+    }
+  ],
+  "type": ["VerifiableCredential"],
+  "issuer": "did:key:z6MkqG......",
+  "issuanceDate": "2023-01-01T00:00:00Z",
+  "credentialSubject": {
+    "id": "did:key:z6Mkh......",
+    "name": "John Doe"
+  },
+  "proof": {
+    "type": "Ed25519Signature2020",
+    "created": "2023-01-01T00:00:00Z",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:key:z6MkqG......#z6MkqG......",
+    "jws": "eyJhbGciOiJFZERTQSJ9..."
```

Comment:
jws is not used in data integrity proofs, please use `proofValue` with a multibase representation of the proof:
https://w3c.github.io/vc-di-eddsa/#example-signed-credential-1

Reply:
Thanks, this have been updated